packages/components/src/stepper/example.md

```diff
@@ -0,0 +1,68 @@
+```jsx
+import { Stepper } from '@woocommerce/components';
+
+const MyStepper = withState( {
+    currentStep: 'first',
+    isComplete: false,
+} )( ( { currentStep, isComplete, setState } ) => {
+    const steps = [
+        {
+            label: 'First',
+            key: 'first',
+        },
+        {
+            label: 'Second',
+            key: 'second',
+        },
+        {
+            label: 'Third',
+            key: 'third',
+        },
+        {
+            label: 'Fourth',
+            key: 'fourth',
+        },
+    ];
+    const currentIndex = steps.findIndex( s => currentStep === s.key );
+
+    if ( isComplete ) {
+        steps.forEach( s => s.isComplete = true );
+    }
+
+	return (
```

Comment:
minor tabbing a bit out of sorts here with the above.

Reply:
Thanks, Timmy!  Updated 😄